libavfilter/qsvvpp.c

```diff
@@ -981,7 +964,7 @@ int ff_qsvvpp_filter_frame(QSVVPPContext *s, AVFilterLink *inlink, AVFrame *picr
         aframe.frame->frame = NULL;
     };
 
-    if (!picref)
+    if (!picref || !propref)
```

Comment:
Is there a possibility that the first frame we received is not from input 0, so that propref is NULL but the picref is not NULL. The function return and the first frame is lost?

Reply:
How about moving propref check to just before calling av_frame_copy_props()?

Reply:
Updated, thanks for your comments.